incubator/istio/templates/NOTES.txt

```diff
@@ -56,7 +56,13 @@ Verifying the ServiceGraph service
 
 Deploy your App!
 
-  kubectl create -f <(istioctl kube-inject -f <your-app-spec>.yaml)
+  If you have Istio Initializer installed, run the following to deploy your App:
```

Comment:
Please conditionalize and print the correct command eg. `{{ if .Values. initializer.enabled }}`

Reply:
Done